target/linux/kirkwood/image/Makefile

```diff
@@ -42,6 +42,13 @@ define Device/checkpoint_l-50
 endef
 TARGET_DEVICES += checkpoint_l-50
 
+define Device/globalscale_sheevaplug
+  DEVICE_TITLE := Globalscale Sheevaplug
+  DEVICE_DTS := kirkwood-sheevaplug
+  BOARD_NAME := sheevaplug
```

Comment:
This is happily copied, but I assume you don't need it for anything?

Reply:
Removed.